packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -4,11 +4,24 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-const fs = require('fs');
-const path = require('path');
-const pluginUtils = require('@rollup/pluginutils');
-const compiler = require('@lwc/compiler');
-const { resolveModule } = require('@lwc/module-resolver');
+import fs from 'fs';
+import path from 'path';
+
+import { Plugin, SourceMapInput } from 'rollup';
+import pluginUtils, { FilterPattern } from '@rollup/pluginutils';
+import { transformSync, StylesheetConfig } from '@lwc/compiler';
+import { resolveModule, ModuleRecord } from '@lwc/module-resolver';
+import { URLSearchParams } from 'url';
+
+export interface RollupLwcOptions {
+    include?: FilterPattern;
+    exclude?: FilterPattern;
+    rootDir?: string;
+    sourcemap?: boolean;
+    modules?: ModuleRecord[];
+    stylesheetConfig?: StylesheetConfig;
+    preserveHtmlComments?: boolean;
```

Comment:
We could add JSDoc comments.

Reply:
Do you mean we can add inline JSDoc style comments for each field that copy the same content from README.md file? Would that be slightly redundant? Also do we generate JSDoc somewhere?

Reply:
> Do you mean we can add inline JSDoc style comments for each field that copy the same content from README.md file? 
Correct. 

> Would that be slightly redundant? Also do we generate JSDoc somewhere?
Tools to automate this exists, but it would require extra setup. The API shape to document is still small, so I think it's fine to copy the `README.md` content in there. If we see it getting out of hand, we can always automate this in the future.